lib/ansible/modules/windows/win_certificate_store.py

```diff
@@ -177,6 +181,15 @@
   become: yes
   become_method: runas
   become_user: SYSTEM
+
+- name: import certificate be used by IIS
+  win_certificate_store:
+    path: C:\Temp\cert.pfx
+    file_type: pkcs12
+    password: StrongPassword!
+    key_exportable: yes
```

Comment:
I thought that IIS didn't require the key to be exportable, just that a key is imported and usable.

Reply:
You are correct that it is not necessary but it is generally recommended in most of the documentation I've read.

Reply:
I just feel it's more of a security issue, I prefer to document that absolute minimum that is needed and not have people open themselves up more.

Reply:
Fair enough. I've removed the line.